src/physics/utilities/solver_config.hpp

```diff
@@ -53,6 +53,16 @@ enum class LinearSolver
   SuperLU
 };
 
+/**
+ * @brief Nonlinear solver type/method
+ */
+enum class NonlinearSolver
+{
+  BasicNewton,
```

Comment:
`MFEMNewton` might be clearer.

Reply:
Thanks!  The best I could come up with was `MFEMDefaultNewton` but I think just `MFEMNewton` is better.